src/main/java/org/reaktivity/nukleus/maven/plugin/internal/ast/AstType.java

```diff
@@ -37,6 +37,10 @@
     public static final AstType VARBYTEUINT32 = new AstType("varbyteuint32");
 
     public static final AstType OCTETS = new AstType("octets");
+    public static final AstType BOUNDED_OCTETS = new AstType("boundedOctets");
+    public static final AstType BOUNDED_OCTETS8 = new AstType("boundedOctets8");
+    public static final AstType BOUNDED_OCTETS16 = new AstType("boundedOctets16");
+    public static final AstType BOUNDED_OCTETS32 = new AstType("boundedOctets32");
```

Comment:
Since these names are internal, suggest `"octets[]"` instead of `"boundedOctets"`, `"octets[8]"` instead of `"boundedOctets8"`, etc.

Reply:
I'm using these names ("boundedOctets8" etc) for some field and method names. If we change the names to `"octets[]"` or `"octets[8]"`, those field and method names will be invalid with `[` and `]` characters.

Reply:
That's fine then, just keep them as `boundedOctets`, etc.